src/widgets/splits/SplitHeader.cpp

```diff
@@ -537,51 +537,69 @@ std::unique_ptr<QMenu> SplitHeader::createMainMenu()
 
-        QObject::connect(moreMenu, &QMenu::aboutToShow, this, [action, this]() {
-            action->setChecked(getApp()->getNotifications()->isChannelNotified(
-                this->split_->getChannel()->getName(), Platform::Twitch));
-        });
-        QObject::connect(action, &QAction::triggered, this, [this]() {
-            getApp()->getNotifications()->updateChannelNotification(
-                this->split_->getChannel()->getName(), Platform::Twitch);
-        });
+            auto notifySeq = h->getDisplaySequence(
+                HotkeyCategory::Split, "setChannelNotification", {{"toggle"}});
+            if (notifySeq.isEmpty())
+            {
+                notifySeq = h->getDisplaySequence(HotkeyCategory::Split,
+                                                  "setChannelNotification",
+                                                  {std::vector<QString>()});
+                // this makes a full std::optional<> with an empty vector inside
+            }
+            action->setShortcut(notifySeq);
 
-        moreMenu->addAction(action);
-    }
+            QObject::connect(
+                moreMenu, &QMenu::aboutToShow, this, [action, this]() {
+                    action->setChecked(
+                        getApp()->getNotifications()->isChannelNotified(
+                            this->split_->getChannel()->getName(),
+                            Platform::Twitch));
+                });
+            QObject::connect(action, &QAction::triggered, this, [this]() {
+                getApp()->getNotifications()->updateChannelNotification(
+                    this->split_->getChannel()->getName(), Platform::Twitch);
+            });
 
-    if (twitchChannel)
-    {
-        auto *action = new QAction(this);
-        action->setText("Mute highlight sound");
-        action->setCheckable(true);
+            moreMenu->addAction(action);
+        }
 
-        QObject::connect(moreMenu, &QMenu::aboutToShow, this, [action, this]() {
-            action->setChecked(getSettings()->isMutedChannel(
-                this->split_->getChannel()->getName()));
-        });
-        QObject::connect(action, &QAction::triggered, this, [this]() {
-            getSettings()->toggleMutedChannel(
-                this->split_->getChannel()->getName());
-        });
+        {
+            auto *action = new QAction(this);
+            action->setText("Mute highlight sounds");
+            action->setCheckable(true);
 
-        moreMenu->addAction(action);
+            auto notifySeq = h->getDisplaySequence(
+                HotkeyCategory::Split, "setHighlightSounds", {{"toggle"}});
+            if (notifySeq.isEmpty())
+            {
+                notifySeq = h->getDisplaySequence(HotkeyCategory::Split,
+                                                  "setHighlightSounds",
+                                                  {std::vector<QString>()});
+                // this makes a full std::optional<> with an empty vector inside
```

Comment:
This was here before, but I don't think it should be here. If you want to keep it, move it one line up (so it's clear what this is referring to).

Reply:
Just to make sure, you're talking about the comment here, and not the whole `empty` branch?

Reply:
Yes, it's about the comment.